workshops/templates/workshops/event.html

```diff
@@ -6,6 +6,17 @@
 {% load tags %}
 
 {% block content %}
+
+{% if event.tags_changed %}
```

Comment:
To the tags as in metatags stored in index.html of the workshop, but I see your point here: there's a name conflict with `Event.tags`.